ethpm_cli/parser.py

```diff
@@ -117,41 +147,49 @@ def add_alias_arg_to_parser(parser: argparse.ArgumentParser) -> None:
 #
 
 
-# todo: extend to pin & release a local manifest
 def release_cmd(args: argparse.Namespace) -> None:
     config = Config(args)
-    release_package(args.package_name, args.package_version, args.manifest_uri, config)
+
+    manifest_uri = args.manifest_uri
+
+    package_name = args.package_name
+    package_version = args.package_version
+
+    if args.manifest_path:
+        (package_name, package_version, manifest_uri) = pin_local_manifest(
+            args.manifest_path
+        )
+        cli_logger.info(
+            f"Retrieving manifest info from local file @ {args.manifest_path} "
+        )
+
```

Comment:
On the python EF team - we try to avoid mutation wherever possible - https://github.com/ethereum/snake-charmers-tactical-manual/issues/27 - If you could update this to use an if/else and avoid mutating `package_name`, `package_version` and `manifest_uri` that'd be better